src/components/FacebookButton.tsx

```diff
@@ -0,0 +1,23 @@
+import FacebookLogin from 'react-facebook-login';
+import { FACEBOOK_APP } from "../constants";
+
+const FacebookButton = () => {
+  const responseFacebook = (response: any) => {
+    console.log(response);
+  }
+
+  return (
+    <div>
+      <FacebookLogin
+        appId={FACEBOOK_APP.id}
+        fields="name,email,picture, user_birthday"
+        scope="public_profile,user_friends, email, user_birthday"
+        callback={responseFacebook}
+        textButton="Continue with Facebook"
+        cssClass=""
```

Comment:
Can you remove 'cssClass=""'? Is there any sense to pass an empty string?

Reply:
Facebook button has own style in applyed library. To not use this style, I can use cssClass property

Reply:
Ok